Import/ImportJob_Globalsat.cs

```diff
@@ -264,114 +268,125 @@ protected void AddActivities(IImportResults importResults, IList<GlobalsatPacket
                     //Insert pauses when estimated/listed distance differs "too much"
                     //Guess pauses - no info of real pause, but this can at least be marked in the track
                     //Share setting with global split
-                    if (detectPausesFromSpeed &&
-                        (foundGPSPoint && activity.GPSRoute.Count > 0 ||
-                        activity.HeartRatePerMinuteTrack.Count > 0))
+                    if (foundGPSPoint && activity.GPSRoute.Count > 0)
                     {
                         //estimated time for the pause
                         double estimatedSec = 0;
-                        //how far from the first point
-                        double perc = 0;
                         string info = "";
-                        bool insertPause = false;
-                        if (activity.GPSRoute.Count > 0)
-                        {
-                            float gpsDist = gpsPoint.DistanceMetersToPoint(activity.GPSRoute[activity.GPSRoute.Count - 1].Value);
-                            //Some limit on when to include pause
-                            //gpsDist must be higher than (all) GPS errors
-                            if (gpsDist > 100 && gpsDist > 3 * dist)
+                        
+                        //speed & cadence method
+                        if (detectPauses == 2 && activity.CadencePerMinuteTrack != null)

```

Comment:
Pause check could be detected also without GPS using the speed track.
A reason to implement this is though that my 580P GPS is broken so I cannot test this as is.